gpgit.sh

```diff
@@ -170,12 +169,11 @@ if [[ "$1" == "-h" || "$1" == "--help" ]]; then
     usage 1>&2
     exit 0
 fi
-shift
 
 # Parse input params an ovrwrite possible default or config loaded options
 GETOPT_ARGS=$(getopt -o "ho:u:e:p:g:w:t:s:m:y" \
             -l "help,output:,username:,email:,project:,gpg:,wget:,tar:,sha:,message:,yes"\
-            -n "$PROGNAME" -- "$@")
+            -n "$PROGNAME" -- $@)
```

Comment:
@cmaglie I think removing the quotes here is not required and also causes problems when passing something with spaces like a message `-m "lol wut"`.

See https://github.com/koalaman/shellcheck/wiki/SC2068